src/builder.rs

```diff
@@ -173,6 +173,26 @@ impl<St: BuilderState> ConfigBuilder<St> {
             .insert(Expression::from_str(key.as_ref())?, value.into());
         Ok(self)
     }
+
+    /// Sets an override if value is set
+    ///
+    /// This function sets an overwrite value if value is not None, otherwise it does nothing.
```

Comment:
Also, don't use the negative case here. Better: "..sets an overwrite value if Some(_) is passed. If None is passed, this function does nothing".